examples/python/aoa.py

```diff
@@ -0,0 +1,97 @@
+# Originally developed by JSBSim Team
+# Modified by Guilherme A. L. da Silva - aerothermalsolutions.co
+# Calculation required by aircraft icing enginering
+
+import jsbsim
+import xml.etree.ElementTree as ET
+import matplotlib.pyplot as plt
+import matplotlib.style as style
+style.use('fivethirtyeight')
+
+# function to change CG in aircraft xml
+# change the directory to the aircraft to be studied
+def changeCG(cgPos,readOnly):
+    tree = ET.parse('../../aircraft/global5000/global5000.xml')
+    root = tree.getroot()
+
+    for x in root.findall('mass_balance/location'):
+        cg = x.find('x').text
+        if not readOnly:
+             x.find('x').text=str(cgPos)
+             tree.write('../../aircraft/global5000/global5000.xml')
+    return cg
```

Comment:
As previously mentioned by @seanmcleod in your PR #435:
> This cg is the cg for the empty weight aircraft with the associated moments of inertia for this empty weight aircraft. So you can't move it without also updating the moments of inertia.

This could be fixed with the [parallel axis theorem](https://en.wikipedia.org/wiki/Parallel_axis_theorem) however.

Reply:
Yes...but it refers to fuel weight only not empty weight.